executor/adapter.go

```diff
@@ -892,14 +893,24 @@ func (a *ExecStmt) SummaryStmt() {
 	planGenerator := func() string {
 		return plannercore.EncodePlan(a.Plan)
 	}
-	_, planDigest := getPlanDigest(a.Ctx, a.Plan)
+	// Generating plan digest is slow, only generate it once if it's 'Point_Get'.
```

Comment:
If not generating plan digest here, then calculate the statement `key.Hash()` in https://github.com/pingcap/tidb/pull/14490/files#diff-b92b32d3c804ceaa9c4cbc5ad8ab0d2aR249
will not contain the plan digest?

Reply:
Addressed.